### PR TITLE
Update tests search runs

### PR DESCRIPTION
**Pull request by ruthfortmann**

### CODE REVIEW

File: mlflow_faculty/tracking.py [hunk 1]
```diff
@@ -352,8 +352,8 @@ def search_runs(
         :return: A list of :py:class:`mlflow.entities.Run` objects that satisfy
             the search expressions
         """
-        if search_filter is not None:
-            raise NotImplementedError("search_expressions must be set to None")
+        if not(filter_string is None or filter_string.strip() == ""):
```
Comment (srstevenson) [inline mlflow_faculty/tracking.py]:
```suggestion
        if filter_string is not None and filter_string.strip() != "":
```

File: mlflow_faculty/tracking.py [hunk 2]
```diff
@@ -334,12 +334,12 @@ def get_metric_history(self, run_id, metric_key):
...
             ]
 
-    def search_runs(
+    def _search_runs(
         self,
         experiment_ids,
-        search_filter=None,
-        run_view_type=None,
-        max_results=None,
+        filter_string,
```
Comment (srstevenson) [inline mlflow_faculty/tracking.py]: The docstring needs updating too.

File: mlflow_faculty/tracking.py (hunk 1 above)
Comment (ruthfortmann) [reply mlflow_faculty/tracking.py]: Yep agreed. updated it

File: mlflow_faculty/tracking.py (hunk 2 above)
Comment (ruthfortmann) [reply mlflow_faculty/tracking.py]: Done

**Comment by acroz** [comment]
Thanks for the contribution @ruthfortmann!

**Pull request merged**